fluid-music/src/tracksToReaperProject.js

```diff
@@ -0,0 +1,191 @@
+const rppp = require('rppp')
+const tab = require('./tab')
+
+// Requires a bpm
+
+
+/**
+ * Create a `ReaperProject` from a `TracksObject`
+ *
+ * ```javascript
+ * const session = fluid.score.parse(myScore, myConfig);
+ * const message = fluid.score.tracksToFluidMessage(session.tracks);
+ * ```
+ *
+ * @param {TracksObject} tracksObject A tracks object generated by score.parse
+ * @returns {ReaperProject}
+ */
+function tracksToReaperProject(tracksObject) {
```

Comment:
For now, can you make bpm be a second argument? so the function signature would be `function tracksToRepaerProject(tracksObject, bpm)`? 

I think it should throw a TypeError if the user doesn't supply a `bpm` number. This will be a good reminder that the API isn't totally finished -- making it more difficult to forget that the bpm is needed when creating a Reaper project.